src/routes.tsx

```diff
@@ -1,29 +1,55 @@
 import { HashRouter, Route, Switch } from "react-router-dom";
-import React from "react";
-import { WalletProvider } from "./contexts/wallet";
+import React, { useMemo } from "react";
+import { WalletProvider } from "@solana/wallet-adapter-react";
 import { ConnectionProvider } from "./contexts/connection";
 import { AccountsProvider } from "./contexts/accounts";
 import { MarketProvider } from "./contexts/market";
 import { AppLayout } from "./components/Layout";
 
 import { FaucetView, HomeView } from "./views";
+import {
+  getLedgerWallet,
+  getMathWallet,
+  getPhantomWallet,
+  getSolflareWallet,
+  getSolletWallet,
+  getSolongWallet,
+} from "@solana/wallet-adapter-wallets";
 
 export function Routes() {
+  const wallets = useMemo(
+    () => [
+      getPhantomWallet(),
+      getSolflareWallet(),
+      // TODO: Get tor.us wallet client Id
+      // getTorusWallet({
+      //   options: {
+      //     clientId: "Go to https://developer.tor.us and create a client ID",
+      //   },
+      // }),
```

Comment:
See if you can try using the clientId in the wallet-adapter repo example.

Reply:
Worked!